docs/whatsnew/6.1.rst

```diff
@@ -23,6 +23,41 @@ By the numbers:
 * X distinct people have contributed code
 
 
+.. _whatsnew-6.1-table-column-order-from-rows:
+
+Ordering of table columns constructed from ``rows``
+===================================================
+
+The order of columns as they appear in a ``Table`` constructed as ``Table(rows=...)``
+may differ between ``astropy`` ``6.0`` and ``6.1``. Specifically, in ``6.0``,
+then column ordering was determined by the first :class:`dict` if it contained values
+for all the columns, or by sorting the column names alphabetically if it did not.
+In ``6.1``, columns are simply inserted as they appear when the :class:`list` of
+:class:`dict` is iterated over.
+
+As an example,
```

Comment:
As a general rule, do not make a single long sentence containing multiple code blocks. Instead make a short sentence or clause in front of each one.
```suggestion
For example, create a table as shown below:
```